README.md

```diff
@@ -42,6 +42,12 @@ by targeting the `/_next/image/*` route:
     X-Test = 'foobar'
 ```
 
+## Disabling `ipx`
```

Comment:
[pebble] I don't like the way this is worded. I don't think ipx is the thing to talk about here, because there's no reason anyone should care or know about what ipx is. The point is that it's disabling the image loader, and that they should only do it if they are either not using `next/image`, or are using a different loader.

Reply:
I've updated the README, let me know if that's a bit better or if you would like to see further changes